### PR TITLE
[W5.11][T12-2] Cheong Chee Mun Brandon

### PR DESCRIPTION
**Pull request by brandonccm1996**

### CODE REVIEW

**Comment by chewkahmeng** [comment]
For making the find command insensitive, maybe you could have used equalsIgnoreCase to compare all the names in the list instead of changing every name to lowercase and comparing them.

File: test/expected.txt [hunk 1]
```diff
@@ -213,8 +213,9 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo Tags: [secretive]
```
Comment (marlenekoh) [inline test/expected.txt:216]: good job at updating tests

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -48,9 +48,17 @@ public CommandResult execute() {
...
+            keywordsLowercase.add(keyword.toLowerCase());
+        }
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            ArrayList<String> wordsInNameLowercase = new ArrayList<>();
+            for (String word : wordsInName) {
+                wordsInNameLowercase.add(word.toLowerCase());
+            }
+            if (!Collections.disjoint(wordsInNameLowercase, keywordsLowercase)) {
```
Comment (marlenekoh) [inline src/seedu/addressbook/commands/FindCommand.java:61]: good, but don't forget to to update javadoc comment for `FindCommand` and `MESSAGE_USAGE` (:

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -79,17 +79,17 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 [NOTE]
 ====
-The search is case sensitive, the order of the keywords does not matter, only the name is searched,
+The search is not case sensitive, the order of the keywords does not matter, only the name is searched,
```
Comment (marlenekoh) [inline docs/UserGuide.adoc:82]: good job at updating documentation

**Review by marlenekoh — COMMENTED** [review]
@brandonccm1996 good job at making a clean PR (:

Please close PR after reading review comments.

**Pull request closed without merging**